### PR TITLE
pick: change log level to debug - Closes #361

### LINKED ISSUE #361: pick function logs are too verbose
## Problem
The `pick()` function logs out `message.data` before and after the transformation, with `log.level=info`.

This causes a problem for me where the logs become almost double the size of the data I'm transferring, which in my case is 100's of GB. I could change `log.level` to `error`, but I need the metric logs, which are also `info` logs, to track progress.

I also realized this logging seems inconsistent with the other functions (`omit`, `rename`, `skip`, etc...) which don't log anything.

### Proposed Options
I'm not sure what the best or right solution is here, but these are some possible options I came up with:
- change the log level from info to debug
- remove addition `msg` property of 'picking...' and '...picked'
- remove logs from pick altogether

I can submit a PR once I get feedback on the right solution.

### PR DESCRIPTION
**Pull request by kiavashps**
### Required for all PRs:

- [ ] CHANGELOG.md updated (feel free to wait until changes have been reviewed by a maintainer)
- [ ] README.md updated (if needed)

### CODE REVIEW

File: function/pick/picker.go [hunk 1]
```diff
@@ -20,13 +20,13 @@ type Picker struct {
 }
 
 func (p *Picker) Apply(msg message.Msg) (message.Msg, error) {
-	log.With("msg", msg).Infof("picking...")
+	log.With("msg", msg).Debugf("picking...")
```
Comment (jipperinbham) [inline function/pick/picker.go]: since it wouldn't seem obvious, you can change this to `Debugln` (same below as well)

**Comment by jipperinbham** [comment]
ignore the failed tests, there's an issue I've been dealing with that fails on travisci but not locally

File: function/pick/picker.go (hunk 1 above)
Comment (kiavashps) [inline function/pick/picker.go]: ok cool!

**Comment by kiavashps** [comment]
@jipperinbham should be good to go!

**Pull request merged**